citar.el

```diff
@@ -222,6 +222,23 @@ If nil, single resources will open without prompting."
   :group 'citar
   :type '(boolean))
 
+(defcustom citar-open-note-functions
+  '(citar-file--open-note citar--open-note-field citar-org-roam-open-note)
+  "List of functions to open a note."
+  :group 'citar
+  :type '(function))
+
+(defcustom citar-create-note-function nil
+  "Function to create a new note."
+  :group 'citar
+  :type 'function)
```

Comment:
Arguably we don't need this; this behavior can be handled in the actual note open-note function?